src/ophys_etl/transforms/suite2p_container.def

```diff
@@ -26,6 +26,7 @@ From: continuumio/miniconda3:4.8.2
     echo "export CONDA_BASE=\"${CONDA_BASE}\"" >> $SINGULARITY_ENVIRONMENT
 
     # install Suite2P
+    apt-get update
```

Comment:
I think this should be `apt-get update && apt-get install -q -y libgl1-mesa-glx` to take advantage of layer caching...